src/argus_htmx/tailwindtheme/snippets/20-loading-box.css

```diff
@@ -1,18 +1,25 @@
 @layer components {
+   /*
+    * Example usage:
+    * <div class="loading-box">
+    *   <!-- Any content tag here -->
+    *   <div class="htmx-indicator">
+    *     <!-- Any optional feedback widget here -->
+    *   </div>
+    * </div>
+    */
     .loading-box {
-      position: relative;
+      display: grid;
+      place-items: center;
     }
-    .loading-box .loading-overlay {
-      position: absolute;
-      top: 0;
-      left: 0;
-      height: 100%;
-      width: 100%;
-      display: flex;
-      align-items: center;
-      justify-content: center;
+    /* Stacks direct children on top of each other */
+    .loading-box > * {
+      grid-column-start: 1;
+      grid-row-start: 1;
     }
-    .loading-box .loading-overlay:not(.htmx-request) {
-      display: none;
+    /* Adds overlay */
+    .loading-box:has(> .htmx-indicator.htmx-request) > *:not(.htmx-indicator) {
+      pointer-events: none;
+      opacity: 0.3;
     }
```

Comment:
ah, I guess `pointer-events: none;` make clicks "pass through" to the underlying element? That's nice. I had seen it before but never given it much thought :)